### ISSUE TITLE
A feature request for the Latin version only

### ISSUE THREAD

**Issue by Antares-LYA** [JulietaUla/Montserrat#94]
I was wondering if it's possible to add a Latin-only version of the font? If using it for the web, every kilobyte counts, and due to all the extra languages symbols support its size is almost double of other fonts, like Roboto or Lato. It's great for the desktop purposes, but when it's used in a web design and I know there's no need for the non-Latin characters support, it's extra 100 kilobytes or so of the data that won't be used.

**Comment by alerque** [comment]
For the web Google fonts already breaks it up into pieces by character range. You can do the same thing yourself if you are self-hosting fonts and extract just whatever subset is useful to you. This isn't something most upstream font projects need to worry about.

**Comment by alerque** [comment]
You probably don't want to use the source to re-export the font, you are liable to end up with sub-optimal settings somewhere. Just chopping up an existing export is probably a better idea.

**Comment by Antares-LYA** [comment]
Yeah, I wasn't using the source. Found a nice guide at the web on how to remove the unwanted glyphs from the font file in FontForge, kept only the default typography: english letters, digits, punctuation, etc.  Ended up with .ttf 49 KB vs. 269 KB original and .woff files around 20 KB vs. around 90 KB original.

**Comment by karlrudd** [comment]
A good online option for font subsetting is the Webfont Generator at Font Squirrel: https://www.fontsquirrel.com/tools/webfont-generator

**Comment by Antares-LYA** [comment]
Thanks a lot, another great option! I've tried it, and it's really easy to use.
